mailserver.env

```diff
@@ -346,6 +346,13 @@ POSTFIX_REJECT_UNKNOWN_CLIENT_HOSTNAME=0
 # Note: More details at http://www.postfix.org/postconf.5.html#inet_protocols
 POSTFIX_INET_PROTOCOLS=all
 
+# If enabled, STARTTLS support is enforced for outgoing mails to domains
+# with MTA-STS records like Google Mail.
+# This can prevent man-in-the-middle that hide the STARTTLS feature.
+# - **0** ==> MTA-STS disabled
+# - 1 => MTA-STS enabled
+ENABLE_MTA_STS=0
```

Comment:
We have a similar page in the docs that should be updated as well.

While this is only outbound support for MTA-STS, it may warrant a bit more documentation.
- [Configuration for the daemon service](https://github.com/Snawoot/postfix-mta-sts-resolver#mta-sts-daemon-configuration). Defaults should be fine, so only mentioning it in this PR may be fine.
- [Configuration of Postfix](https://github.com/Snawoot/postfix-mta-sts-resolver#postfix-configuration), this PR covers that and we already configure the CA path location. Nothing needs to be covered regarding that.
- [DANE compatibility](https://github.com/Snawoot/postfix-mta-sts-resolver#warning-mta-sts-policy-overrides-dane-tls-authentication), this may be worth making users aware of.

Thus **minimal requirement** is to document this setting in ENV docs page, and reference the DANE concern.

Documenting support for MTA-STS inbound via policy file and DNS record (_neither requiring integration with DMS, but might be of interest to users interested in the feature for our docs to cover_) would be nice, but not a requirement.

Reply:
I'll work on the documentation side of thing.